test/e2e/framework/pv/pv.go

```diff
@@ -621,6 +624,7 @@ func MakePersistentVolumeClaim(cfg PersistentVolumeClaimConfig, ns string) *v1.P
 
 	return &v1.PersistentVolumeClaim{
 		ObjectMeta: metav1.ObjectMeta{
+			Name:         cfg.Name,
 			GenerateName: cfg.NamePrefix,
```

Comment:
I think `GenerateName` should not be set if `cfg.Name` is set?

Reply:
My understanding is that this MakePersistentVolumeClaim() is shared.  Some callers will set Name while other will set GenerateName only.  Is your suggestion to add some checks here to make sure only one is set?

Reply:
Ah I see, ignore my comment then. I had thought that setting both Name and GenerateName could be problematic, but seems like if you set Name, the API will ignore GenerateName anyways so this seems fine.